physionet-django/console/templates/console/user_management.html

```diff
@@ -35,6 +35,30 @@ <h3>Profile</h3>
       {{ subject.is_credentialed }}
     </div>
   </div>
+  <div class="row mb-1">
```

Comment:
Later we might want to move the cloud details into a separate subsection, but this is fine for now.